src/sentry/static/sentry/app/views/alerts/list/header.tsx

```diff
@@ -47,45 +46,49 @@ const AlertHeader = ({router, organization, activeTab}: Props) => {
             }
             type="beta"
           />
-        </StyledPageHeading>
-
+        </StyledLayoutTitle>
+        <StyledNavTabs underlined>
+          <Feature features={['incidents']} organization={organization}>
+            <li className={activeTab === 'stream' ? 'active' : ''}>
+              <Link to={`/organizations/${organization.slug}/alerts/`}>
+                {t('Stream')}
+              </Link>
+            </li>
+          </Feature>
+          <li className={activeTab === 'rules' ? 'active' : ''}>
+            <Link to={`/organizations/${organization.slug}/alerts/rules/`}>
+              {t('Rules')}
+            </Link>
+          </li>
+        </StyledNavTabs>
+      </StyledLayoutHeaderContent>
+      <Layout.HeaderActions>
         <Actions gap={1}>
-          <Button
-            onClick={handleNavigateToSettings}
-            href="#"
-            size="small"
-            icon={<IconSettings size="xs" />}
-          >
+          <Button onClick={handleNavigateToSettings} href="#" icon={<IconSettings />}>
             {t('Settings')}
           </Button>
 
           <CreateRuleButton organization={organization} router={router} />
         </Actions>
-      </PageHeader>
-      <StyledNavTabs underlined>
-        <Feature features={['incidents']} organization={organization}>
-          <li className={activeTab === 'stream' ? 'active' : ''}>
-            <Link to={`/organizations/${organization.slug}/alerts/`}>{t('Stream')}</Link>
-          </li>
-        </Feature>
-        <li className={activeTab === 'rules' ? 'active' : ''}>
-          <Link to={`/organizations/${organization.slug}/alerts/rules/`}>
-            {t('Rules')}
-          </Link>
-        </li>
-      </StyledNavTabs>
-    </React.Fragment>
+      </Layout.HeaderActions>
+    </Layout.Header>
   );
 };
 
 export default AlertHeader;
 
-const StyledPageHeading = styled(PageHeading)`
-  display: flex;
-  align-items: center;
+const StyledLayoutHeaderContent = styled(Layout.HeaderContent)`
+  margin-bottom: 0;
+`;
+
+const StyledLayoutTitle = styled(Layout.Title)`
+  margin-top: 0;
 `;
 
 const StyledNavTabs = styled(NavTabs)`
+  margin-top: 15px;
+  margin-bottom: 0;
+  border-bottom: 0 !important;
```

Comment:
Why do we zero out these margins if we're trying to match Discover's layout

Reply:
discover isn't using a tab navigation here, they have a title instead.

Reply:
Yeah, we should build tabs into the Layout component eventually so we don’t have to do all this styled stuff.